tests/lib/test_scheduler.py

```diff
@@ -1,12 +1,13 @@
 # SPDX-License-Identifier: BSD-2-Clause
 # torii: UnusedElaboratable=no
 
-from torii.hdl           import *
 from torii.asserts       import *
-from torii.sim           import *
+from torii.hdl           import *
 from torii.lib.scheduler import *
+from torii.sim           import *
+
+from ..utils import ToriiTestSuiteCase
```

Comment:
Same comment about re-spacing `import` here

Reply:
oops, missed this one, sorted